test/index.spec.js

```diff
@@ -70,6 +70,27 @@ describe('systemjs-plugin-ko-tpl', function () {
                     );
                 });
             });
+            it('should insert a nested template into <head> as separate script tags', function () {
+                var fakeTemplate = [
+                    '<script type="text/html" id="nestedTemplateOne">',
+                    '    <h1>NESTED TEMPLATE ONE</h1>',
+                    '</script>',
+                    '<script type="text/html" id="nestedTemplateTwo">',
+                    '    <h1>NESTED TEMPLATE TWO</h1>',
+                    '</script>'
+                ].join('\n') + '\n';
+                var fakeContext = { builder: false };
+                var load = loadFactory('fakeTemplate.ko');
+                var fetch = function () { return Promise.resolve(fakeTemplate); };
+                var promise = plugin.fetch.call(fakeContext, load, fetch);
+                return expect(promise, 'to be fulfilled').then(function () {
```

Comment:
I generally like wrapping Promises in an expect call like here. I don't know if it really makes a whole lot of a difference, but it feels right... :feelsgood: 

The only difference is that using this extra wrapping gives me a pretty unexpected error telling me that it expected something to be fulfilled, but it wasnt - where without it, I'd see the rejection reported as an error from mocha.

Is there any reason not to do so, except that it's a bit verbose?